cypress/features/regression/drawer.feature

```diff
@@ -2,42 +2,46 @@ Feature: Drawer component
   I want to test Drawer component
```

Comment:
If we are testing it in `Test` directory this file also needs to be moved to `Test` in `cypress`

Reply:
fixed